projects.py

```diff
@@ -0,0 +1,136 @@
+                project_bill_code varchar(10),
+                primary key (id)
+            );
+        '''.format(db=args_dict['db'], table=args_dict['table']))]
+    elif args_dict['dbengine'] == 'sqlite':
+        sql_commands = [('''
+            create table if not exists {table} (
+                id integer primary key autoincrement,
+                project_name varchar(128),
+                project_code varchar(8),
+                project_bill_code varchar(10)
+            );
+        '''.format(table=args_dict['table']))]
+    else:
+        sys.exit(DB_ERROR_MESSAGE)
+
+    if args_dict['dbengine'] == 'mysql':
+        sql_commands.insert(0, 'create database if not exists {};'.format(args_dict['db']))
+
+    sql_commands.append(('''
+        insert into {table} (project_name, project_code, project_bill_code)
+            values ('MyFirstProject', 'proj1', '111111');
+    '''.format(table=args_dict['table'])))
+
+    return sql_commands
+
+
+def add_projects(args_dict, db):
+    table_check(args_dict, db)
+    sql_commands = [('''
+        insert into {table} (project_name, project_code, project_bill_code)
+            values ("{desc}", "{code}", "{nbr}");
+    '''.format(table=args_dict['table'],
+               desc=args_dict['add'][0],
+               code=args_dict['add'][1],
+               nbr=args_dict['add'][2]))]
+
+    if args_dict['dbengine'] == 'mysql':
+        sql_commands.insert(0, 'use {};'.format(args_dict['db']))
+
+    return sql_commands
+
+
+def check_projects(args_dict, db):
+    table_check(args_dict, db)
+    return pdsql.read_sql('select * from {table}'.format(table=args_dict['table']), db)
+
+def run(args_dict):
+    args_dict = update_args(args_dict)
+    if args_dict['check'] and args_dict['add']:
+        sys.exit('STOP! You can\'t `check` and `add` entries at once.')
+
+    if args_dict['dbengine'] == 'mysql':
+        db = mdb.connect(
+            host='{}'.format(args_dict['host']),
+            user='{}'.format(USERNAME),
+            password='{}'.format(PASSWORD),
+            db=args_dict['db'],
+            autocommit=True
+        )
+    elif args_dict['dbengine'] == 'sqlite':
+        db = sqlite3.connect('{}.db'.format(args_dict['db']), isolation_level=None)
+    else:
+        sys.exit(DB_ERROR_MESSAGE)
+
+    if args_dict['check']:
+        print check_projects(args_dict, db)
+    else:
+        if args_dict['add']:
+            sql_commands = add_projects(args_dict, db)
+        else:
+            sql_commands = create_projects(args_dict)
+
+        for sql in sql_commands:
+            db.cursor().execute(sql)
+        db.cursor().close()
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Add a project table to database.')
+    parser.add_argument('-d', '--db', required=False, help='Name of the database '
+                        'to create.')
+    parser.add_argument('-t', '--table', required=False, help='Name of the table to '
```

Comment:
since `table` in the config file refers to the time tracking table, the table argument should be required, or have a different name than `table` so that it doesn't fall back on the config file value.